book/03-git-branching/sections/rebasing.asc

```diff
@@ -32,60 +28,49 @@ First, rewinding head to replay your work on top of it...
-That way, the maintainer doesn't have to do any integration work -- just a fast-forward or a clean apply.
+Kadalasan, gagawin mo ito upang siguraduhing ang iyong mga commit ay nalapat nang malinis sa isang remote na branch -- marahil sa isang proyekto kung saan sinusubukan mong mag-ambag ngunit iyong hindi mo na pinapanatili. Sa kasong ito, gagawin mo ang iyong trabaho sa isang branch at pagkatapos ay iri-rebase ang iyong trabaho sa `origin/master` kapag ikaw ay handa nang isumite ang iyong mga patch sa pangunahing proyekto. Sa paraan iyon, ang tagapanatili ay hindi na kailangang gumawa ng anumang pagsasama na trabaho -- isang fast-forward o isang malinis na paglapat lamang.
 
-Note that the snapshot pointed to by the final commit you end up with, whether it's the last of the rebased commits for a rebase or the final merge commit after a merge, is the same snapshot – it's only the history that is different.
-Rebasing replays changes from one line of work onto another in the order they were introduced, whereas merging takes the endpoints and merges them together.
+Tandaan na ang snapshot na itinuro ng huling commit na nagtapos sa iyo, kung ito ay ang huli sa na-rebase na mga commit para sa isang rebase o ang huling merge na commit pagkatapos ng isang merge, ay ang parehong snapshot - ang kasaysayan lamang ang naiiba. Ang pag-rebase ay iri-replay ang mga pagbabago mula sa isang linya ng trabaho patungo sa iba sa pagkakaayos base sa kanilang pagpakilala, samantalang ang pag-merge ay kinukuha ang mga endpoint at sama-samang imi-merge ang mga ito.
 
-==== More Interesting Rebases
+==== Maraming Kagiliw-giliw na mga Rebase
 
-You can also have your rebase replay on something other than the rebase target branch.
-Take a history like <<rbdiag_e>>, for example.
-You branched a topic branch (`server`) to add some server-side functionality to your project, and made a commit.
-Then, you branched off that to make the client-side changes (`client`) and committed a few times.
-Finally, you went back to your server branch and did a few more commits.
+Maaari ka ring magkaroon ng iyong rebase na replay sa anuman maliban sa rebase na target na branch. Gamitin ang isang kasaysayan katulad ng <<rbdiag_e>>, bilang halimbawa. Ikaw ay nag-branch ng isang paksa na branch (`server`) upang magdagdag ng ilang server-side na functionality sa iyong proyekto, at gumawa ng isang commit. Pagkatapos, ikaw ay nag-branch off nito upang gawin ang mga pagbabago sa client-side (`client`) at nag-commit ng ilang beses. Sa wakas, ikaw ay bumalik sa iyong server na branch at gumawa ng ilang mga commit.
 
 [[rbdiag_e]]
-.A history with a topic branch off another topic branch
-image::images/interesting-rebase-1.png[A history with a topic branch off another topic branch.]
+.Isang history na may isang paksa na naka-branch off sa ibang paksa na branch
```

Comment:
history  to kasaysayan

Reply:
Will update this now. Thank you @robin-ho.